db/migrations/create_chitter_table.sql

```diff
@@ -1,3 +1,4 @@
 CREATE DATABASE chitter;
 CREATE TABLE peeps(id SERIAL PRIMARY KEY, peeps VARCHAR(60));
--- ALTER TABLE peeps ADD COLUMN title VARCHAR(60);
\ No newline at end of file
+ALTER TABLE peeps ADD COLUMN timestamp timestamp with time zone;
```

Comment:
Nice, I did not think about the time zone. Mine is just the time of post, did not know you could do this.